app/experimenter/kinto/tasks.py

```diff
@@ -92,6 +92,12 @@ def nimbus_check_kinto_push_queue():
             application
         ).first():
             nimbus_end_experiment_in_kinto.delay(queued_end_experiment.id)
+        elif queued_pause_experiment := NimbusExperiment.objects.pause_queue(
+            application
+        ).first():
+            records = {r["id"]: r for r in kinto_client.get_main_records()}
+            record = records[queued_pause_experiment.slug]
```

Comment:
Oh I would do all this in the task and just pass the experiment id in to make this more consistent with the other task signatures